tools

```diff
@@ -1 +1 @@
-Subproject commit 2429dbe36100a9e259b90dbef26303e720afd101
+Subproject commit 070bee3e30dac3de63f52da22bcc278d2911ef99
```

Comment:
in general, I think you're not expected to push the results of the changes made by "git submodule update" to this repo.

In this particular case, I guess the best is probably to revert that part of commit ?